src/keyboard.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <GL/glew.h>
+#include <GLFW/glfw3.h>
+#include <glm/glm.hpp>
+
+/*
+	Handles GLFW calls for getting Mouse and Keyboard Input
```

Comment:
Inaccurate - it handles calls for getting input AND manipulation of the camera in response to the input. Please decouple.